src/Workspaces/Core/Portable/Rename/ConflictEngine/ConflictResolver.cs

```diff
@@ -234,7 +235,7 @@ private static async Task AddDeclarationConflictsAsync(
     {
         try
         {
-            var projectOpt = conflictResolution.CurrentSolution.GetProject(renamedSymbol.ContainingAssembly, cancellationToken);
+            var projectOpt = conflictResolution.CurrentSolution.GetProject(projectId);
```

Comment:
The `GetProject` method that takes an assembly symbol uses the compilation tracker to work, and with source generators the compilation tracker stuff gets odd. Fortunately, the one caller of this method already has the project Id to hand, so it was easiest to just pass it in.

Reply:
basically, i love this.  but needs a bit of an audit just to make firmly sure this project id is what we expect.